core-requirements.txt

```diff
@@ -1,6 +1,6 @@
 numpy>=1.20.0
 pandas>=1.2.1
-scipy>=1.3.3
+scipy>=1.5.0
```

Comment:
Added because scipy changed how they return values for the Shapiro test (as a named tuple)